parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -343,7 +352,6 @@ ensureK8sControlPlane() {
     if $REBOOTREQUIRED || [ "$NO_OUTBOUND" = "true" ]; then
         return
     fi
-    wait_for_file 3600 1 $KUBECTL || exit $ERR_FILE_WATCH_TIMEOUT
```

Comment:
We don't need to wait for this file as `retrycmd_if_failure` will handle the error of the file not existing for us